{{cookiecutter.project_slug}}/.drone.yml

```diff
@@ -21,7 +21,7 @@ steps:
     path: ${PRE_COMMIT_HOME}
   commands:
   - export PRE_COMMIT_HOME=$CI_PROJECT_DIR/.cache/pre-commit
-  - pip install -q pre-commit
+  - {{ "-q pre-commit"|install(cookiecutter)}}
```

Comment:
So that's rendered out to either `pip install`, or the package manager equivalent, when the template is rendered?